tree/10_ncf_internals/initialization.cf

```diff
@@ -0,0 +1,44 @@
+#####################################################################################
+# Copyright 2016 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# Initialization
+# This bundle will make necessary step to prepare ncf before running any configuration
+# It needs to be called just after loading ncf and its configuration.
+bundle agent init
+{
+
+  files:
+      "${sys.workdir}/modules/"
+        copy_from         => local_dcp("${me.framework_path}/10_ncf_internals/modules/"),
```

Comment:
`me.framework` is defined as `${sys.workdir}/inputs` so that's no going to work as is.

There is also `${this.promise_dirname}` that could be good to get this. I propose that you make a relative path from there.